onboarding/src/static/css/EmployeeAnswersView.scss

```diff
@@ -50,8 +53,8 @@
     position: fixed;
     /* Height of navbar: 56px */
     top: 56px;
-    /* Width of left menu: 239px */
-    left: 239px;
+    /* Width of left menu: 279px */
+    left: 279px;
```

Comment:
How or why the difference of `40` pixels is so important? :slightly_smiling_face: